### PR TITLE
IIIF drag and drop added to Mirador show page

### LINKED ISSUE #680: Add IIIF badge to show page for objects w/ a manifest
Some information about IIIF drag and drop here:

http://zimeon.github.io/iiif-dragndrop/

Design in #679

### LINKED ISSUE #693: Non-image items with a custom manifest URL still show a viewer
Introduced in #668 

We need some better way not to show a viewer when there is a custom pattern.

### PR DESCRIPTION
**Pull request by mejackreed**
Closes #680 

Also closes #693 

I'm not sure why we have to drag twice. Is this an issue w/ mirador? cc @aeschylus (I remember something like this from before)

dnd

### CODE REVIEW

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -13,4 +13,12 @@ def document_collection_title(value:, **)
   def document_leaflet_map(document:, **)
     render_document_partial(document, 'show_leaflet_map_wrapper')
   end
+
+  ##
+  # @param [String] manifest
+  def iiif_drag_n_drop(manifest)
+    link_to mirador_index_path(manifest: manifest), class: 'iiif-dnd pull-right', data: { turbolinks: false } do
```
Comment (jkeck) [inline app/helpers/application_helper.rb]: I _think_ we want the base of the link to go to an information page about IIIF on library.stanford.edu that was described in #679 (but maybe wasn't explicitly called out as acceptance criteria in the issue.

**Comment by jkeck** [comment]
@mejackreed what's the manifest URL you're using that you're seeing the double drag-and-drop behavior?  I seem to remember that being something to do w/ a miss-match in the url and the `@id` in the manifest.

File: app/helpers/application_helper.rb (hunk 1 above)
Comment (ggeisler) [reply app/helpers/application_helper.rb]: Yes, we want a click to go to this page:

https://library.stanford.edu/projects/international-image-interoperability-framework/viewers

We might want to update that page but I think it is good enough to serve the purpose as is.

Comment (mejackreed) [reply app/helpers/application_helper.rb]: Thanks for catching that. I've updated the PR to do that.

**Comment by mejackreed** [comment]
@jkeck looks like its our indexing code that is indexing `manifest.json`.. creating an issue there

**Pull request merged**